app/services/post_alerter.rb

```diff
@@ -624,6 +619,11 @@ def notify_pm_users(post, reply_to_user, notified)
     end
   end
 
+  def group_notifying_via_smtp(post)
+    return nil if !SiteSetting.enable_smtp || !SiteSetting.enable_imap || post.post_type != Post.types[:regular]
```

Comment:
Sorry I moved this method at the same time...change is adding `!SiteSetting.enable_imap` and `imap_enabled: true` to the group query